### PR TITLE
Adding SDK version to the operator

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
As for creating the operator bundle we need to know which version of
 SDK should be used to generate the manifests

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -47,9 +47,10 @@ func init() {
 }
 
 func printVersion() {
-	log.Info(fmt.Sprintf("Operator Version: %s", version.Operator))
-	log.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
-	log.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
+	log.Info(fmt.Sprintf("version: v%s", version.VersionOperator))
```
Comment (jottofar) [inline main.go:50]: Why the change from `Operator Version`?

Comment (LalatenduMohanty) [reply main.go:50]: You mean from `version.Operator` to `version.versionOperator` ? to keep it consistent with `Version.OperatorSDK`.

Comment (jottofar) [reply main.go:50]: Why did you change the label string from "Operator Version:" which seems more clear?

Comment (LalatenduMohanty) [reply main.go:50]: Will change ti to `operator version`

Comment (wking) [reply main.go:50]: I am in favor of the `Makefile` change, but would rather keep this out of Go, see [here][1] and that commit message.  Is there a benefit to storing this information in Go?

[1]: https://github.com/openshift/cincinnati-operator/commit/c95c673082138ce8a6efb92b789791a952e4dde4#diff-f6bf0ae0d21c9408c6624174da11dafb369ce306f2cdcea704285d8acc87e19eL5

File: main.go [hunk 2]
```diff
@@ -47,9 +47,10 @@ func init() {
...
+	log.Info(fmt.Sprintf("operator-sdk version: v%s", version.VersionSDK))
```
Comment (wking) [inline main.go:51]: I am in favor of the `Makefile` change, but would rather keep this out of Go, see [here][1] and that commit message.  Is there a benefit to storing this information in Go?

[1]: https://github.com/openshift/cincinnati-operator/commit/c95c673082138ce8a6efb92b789791a952e4dde4#diff-f6bf0ae0d21c9408c6624174da11dafb369ce306f2cdcea704285d8acc87e19eL5

File: main.go [hunk 3]
```diff
@@ -47,9 +47,10 @@ func init() {
...
+	log.Info(fmt.Sprintf("go Version: %s", runtime.Version()))
+	log.Info(fmt.Sprintf("go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
```
Comment (wking) [inline main.go:53]: Why the downcasing?  The Go project name is canonically title case, e.g. see examples [here][1].

[1]: https://golang.org/doc/
